ios/FluentUI/Navigation/NavigationController.swift

```diff
@@ -32,7 +32,7 @@ open class NavigationController: UINavigationController {
         return nil
     }
     open override var preferredStatusBarStyle: UIStatusBarStyle {
-        return msfNavigationBar.style == .system ? .default : .lightContent
+        return (msfNavigationBar.style == .system || msfNavigationBar.style == .gradient) ? .default : .lightContent
```

Comment:
This will only look good if our gradient consists of "light" colors. If this is our intention, we should document this somewhere.

Reply:
It's an intentional change requested by partner designers. Clients can always change it for different color combinations if they need to.